src/spikeinterface/comparison/paircomparisons.py

```diff
@@ -704,6 +745,10 @@ class TemplateComparison(BasePairComparison, MixinTemplateComparison):
         List of units from sorting_analyzer_1 to compare.
     unit_ids2 : list, default: None
         List of units from sorting_analyzer_2 to compare.
+    name1 : str, default: "sess1"
+        Name of first session.
+    name2 : str, default: "sess1"
```

Comment:
Shouldn't this be sess2. Seems like a bug to have them both be the same.

Reply:
You are correct